sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -432,6 +432,35 @@ public enum ServiceBusTransportType
         AmqpWebSockets = 1,
     }
 }
+namespace Azure.Messaging.ServiceBus.Amqp
```

Comment:
Based on the PR description, is this just out of date?

Reply:
Already outdated as of comment.